README.md

```diff
@@ -0,0 +1,44 @@
+## Manim DSL Compiler
+
+ManimDSL is a domain specific language using Manim to create animations for
+data structures and algorithms
+
+## Table of Contents:
+- [Installation](#installation)
+- [Usage](#usage)
+- [Documentation](#documentation)
+- [Help with Manim](#help-with-manim)
+- [Contributing](#contributing)
+
+## Installation
+
+ManimDSL produces python code that needs to run on manim to generate the mp4 files. Please  follow the
+[documentation](https://manimce.readthedocs.io/en/latest/installation.html)
+and follow the instructions according to your operating system.
+
+## Usage
+
+Here is an example on how to use ManimDSL:
+
+```js
+    let stack = new Stack;
+    stack.push(1);
+    stack.pop();
+```
+
+Save this file and run the compiler on it:
```

Comment:
Some information on how to obtain `manimdsl.jar` would be good

Reply:
Yeah I agree, we just don't know how that will be yet, including docker still too

Reply:
I will add this in when we have a clearer idea of releases and docker image pulling